lib/content-api-client.js

```diff
@@ -0,0 +1,99 @@
+require('colors');
+const NetworkUtils = require('./utils/NetworkUtils');
+const {
+    renderedRegionsByPageTypeQuery,
+    renderedRegionsByPageTypeAndEntityIdQuery,
+} = require('./graphql/query');
+
+const networkUtils = new NetworkUtils();
+
+/**
+ * @param {object} options
+ * @param {string} options.accessToken
+ * @param {string} options.storeUrl
+ * @param {string} pageType
+ * @returns {Promise<renderedRegions: array}>}
+ */
+async function getRenderedRegionsByPageType({ accessToken, storeUrl, pageType }) {
+    try {
+        const query = renderedRegionsByPageTypeQuery(pageType);
+
+        const response = await networkUtils.sendApiRequest({
+            url: `${storeUrl}/graphql`,
+            headers: {
+                'cache-control': 'no-cache',
+                'content-type': 'application/json',
+                Authorization: `Bearer ${accessToken}`,
+            },
+            method: 'POST',
+            data: JSON.stringify({
+                query,
+            }),
+        });
+
+        const {
+            site: { content },
```

Comment:
is `content` variable for readability? 
It's possible to do like this: 
```suggestion
            const { site: { regions: { renderedRegionsByPageType } } } = response.data.data
```